### ISSUE TITLE
I would like to thank you for your interesting work. How to identify the causes of violence？ Thanks again for your interesting work.

### ISSUE THREAD

**Issue by lawoyiba** [JimLee1996/AVSS2019#7]

**Comment by JimLee1996** [comment]
please see https://github.com/JimLee1996/AVSS2019/tree/master/src/Demo

**Comment by lawoyiba** [comment]
Is it through this line of code? 
def predict(self, x):
        x = x.to(self.device)
        # need N x C x T x H x W
        if len(x.shape) < 5:
            x = x.unsqueeze(0)
        y = self.net(x)

        _, results = y.topk(1, 1, True)

        labels = ['normal' if x else 'violent' for x in results]

        return labels

**Comment by JimLee1996** [comment]
Yep

**Comment by lawoyiba** [comment]
Thank you for your reply. I read your paper. What are the methods used to distinguish violent behavior from normal behavior? Thank you again for your work

**Comment by JimLee1996** [comment]
We use deep learning methods to do this job. 
3DCNNs are proved to be effective on learning spatio-temporal features.
Here we introduce DenseNet architecture to better learn the motion pattern of violent behavior. 
The outputs of model are logits of classes (in this task, violence and non-violence)

**Comment by lawoyiba** [comment]
Thank you for your reply,
After the video passes the DenseNet model, it will identify whether there is violence in the video.

**Comment by JimLee1996** [comment]
Use softmax. Please refer to https://github.com/JimLee1996/AVSS2019/issues/2.

**Comment by lawoyiba** [comment]
Thank you for your reply
